tests/smoke-tests/test-staking-consistency.ts

```diff
@@ -213,4 +215,37 @@ describeSmokeSuite(`Verify staking consistency`, { wssUrl, relayWssUrl }, (conte
 
     debug(`Verified ${checks} lessTotal (runtime: ${specVersion})`);
   });
+
+  it("candidatePool matches candidateInfo", async function () {
+    this.timeout(120000);
```

Comment:
If this long timeout is largely because of the number of candidates, you could make it dependent on `candidatePool`s length.